pkg/reconciler/v1alpha1/service/service.go

```diff
@@ -183,14 +183,14 @@ func (c *Reconciler) reconcile(ctx context.Context, service *v1alpha1.Service) e
 		}
 		c.Recorder.Eventf(service, corev1.EventTypeNormal, "Created", "Created Configuration %q", configName)
 	} else if err != nil {
-		logger.Errorf("Failed to reconcile Service: %q failed to Get Configuration: %q; %v", service.Name, configName, zap.Error(err))
+		logger.Errorf("Failed to reconcile Service: %q failed to Get Configuration: %q; %v", service.Name, configName, err)
```

Comment:
Do we want to use `Errorw` + `Sprintf` here for consistency?

Reply:
+1